### PR TITLE
Ins 2692 move get pending to proc

### PR DESCRIPTION
**Pull request by ivanshibitov**
**- What I did**
1) launch queueProcessor for AdditionalCallFromPreviousExecutor
2) launch UnsafeGetLedgerPendingRequest in flow

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: logicrunner/logicrunner_unit_test.go [hunk 1]
```diff
@@ -185,27 +185,13 @@ func (suite *LogicRunnerTestSuite) TestHandleAdditionalCallFromPreviousExecutor(
...
-			expectedClarifyPendingStateCtr: 1,
 			expectedStartQueueProcessorCtr: 1,
 		},
-		{
-			name:                           "Both procedures fail",
-			clarifyPendingStateResult:      fmt.Errorf("ClarifyPendingState failed"),
-			startQueueProcessorResult:      fmt.Errorf("StartQueueProcessorIfNeeded failed"),
-			expectedClarifyPendingStateCtr: 1,
-			expectedStartQueueProcessorCtr: 0,
-		},
```
Comment (afiskon) [inline logicrunner/logicrunner_unit_test.go]: Test should be fixed, not removed. Or rewritten according to the changes.

Comment (afiskon) [reply logicrunner/logicrunner_unit_test.go]: Oops never mind. I though it's another test.

File: logicrunner/executionstate.go [hunk 2]
```diff
@@ -56,6 +56,7 @@ func NewExecutionState(ref insolar.Reference) *ExecutionState {
 		Ref:         ref,
 		CurrentList: NewCurrentExecutionList(),
 		Queue:       make([]Transcript, 0),
+		pending:     message.PendingUnknown,
```
Comment (ruz) [inline logicrunner/executionstate.go]: this is default, what the point?

File: logicrunner/handle_calls.go [hunk 3]
```diff
@@ -57,6 +57,7 @@ func (h *HandleCall) sendToNextExecutor(ctx context.Context, es *ExecutionState,
 		ObjectReference: es.Ref,
 		Parcel:          parcel,
 		Request:         request,
+		Pending:         es.pending,
```
Comment (ruz) [inline logicrunner/handle_calls.go]: this is wrong. It should be like this:
`
if es.pending == Unknown {
  msg.Pending = NotInPending
} else {
  msg.Pending = es.pending
}
`

File: logicrunner/logicrunner_unit_test.go (hunk 1 above)
Comment (ruz) [reply logicrunner/logicrunner_unit_test.go]: please return tests back

**Pull request merged**